BUILDING.md

```diff
@@ -104,6 +104,8 @@ Prerequisites:
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
 
+Please note that vcbuild is a batch file provided with the Node.js source code. If using PowerShell, use local syntax. For example: ./vcbuild nosign
```

Comment:
I wouldn't know what local syntax means in that context. I'd prefer _use a leading `.\`_.